js-envs/src/main/scala/org/scalajs/jsenv/rhino/RhinoJSEnv.scala

```diff
@@ -36,74 +37,112 @@ final class RhinoJSEnv private (
     semantics: Semantics,
     withDOM: Boolean,
     val sourceMap: Boolean
-) extends ComJSEnv {
+) extends LinkingUnitComJSEnv {
 
   import RhinoJSEnv._
 
-  def this(semantics: Semantics = Semantics.Defaults, withDOM: Boolean = false) =
+  def this(semantics: Semantics, withDOM: Boolean = false) =
     this(semantics, withDOM, sourceMap = true)
 
   def withSourceMap(sourceMap: Boolean): RhinoJSEnv =
     new RhinoJSEnv(semantics, withDOM, sourceMap)
 
+  val symbolRequirements = Emitter.symbolRequirements(semantics, ESLevel.ES5)
```

Comment:
Comment on this, why this is OK. (Non trivial, `RhinoJSEnv` does not use the `Emitter` directly).